src/ui/DynamicFacetManager/DynamicFacetManager.ts

```diff
@@ -71,7 +72,20 @@ export class DynamicFacetManager extends Component {
      */
     compareFacets: ComponentOptions.buildCustomOption<IDynamicFacetManagerCompareFacet>(() => {
       return null;
-    })
+    }),
+    /**
+     * The maximum number of expanded facets inside the manager.
+     * Remaining facets are collapsed.
+     *
+     * **Note:**
+     * Prioritizes facets with active values, and then prioritizes first facets.
+     * If the number of facets with active values exceeds the value of the `maximumNumberOfExpandedFacets` option, it overrides the option.
+     *
+     * Using the value `-1` will disable the feature and keep all facet expanded.
```

Comment:
@jpmarceau added this sweet line

Reply:
Thanks for letting me know :)
will disable -> disables
facet -> facets